lib/backbeat/Metrics.js

```diff
@@ -96,6 +97,18 @@ class Metrics {
         return cb(null, data);
     }
 
+    /**
+     * Uptime of server based on this._internalStart up to max of expiry
+     * @param {number} expiry - max expiry
+     * @return {number} uptime of server up to expiry time
+     */
+    _getMaxUptime(expiry) {
+        let secondsSinceStart = (Date.now() - this._internalStart) / 1000;
+        // allow only a minimum value of 1 for uptime
+        if (secondsSinceStart < 1) secondsSinceStart = 1;
```

Comment:
I think the style guidelines advocate to always use curly braces and put the conditional statement(s) in this block on separate lines.